editoast/src/views/rolling_stocks/light_rolling_stock.rs

```diff
@@ -42,11 +41,38 @@ pub struct LightRollingStock {
     #[schema(required)]
     pub power_restrictions: HashMap<String, String>,
     pub energy_sources: Vec<EnergySource>,
-    #[serde(skip)]
```

Comment:
Why was this removed?

Reply:
I had a clippy error like `clippy error: field `version` is never read`

Reply:
In this case removing the field seems more fitting to me.

Reply:
Done.